### ISSUE TITLE
Cancel image processing

### ISSUE THREAD

**Issue by jsmucr** [darktable-org/darktable#16868]
Since there's a new option to make DT render a preview in export quality, it would also be nice to have an option to cancel such action in progress.

On my PC, rendering a 80 Mpx image employing -- amongst others -- an instance of the Diffuse and sharpen module takes quite some time. I'm talking tens of minutes. When it starts, it cannot be stopped, and it renders the computer unusable for anything else.

Would it be possible to add a button to cancel rendering?

**Comment by jenshannoschwalm** [comment]
Good point. I think it should be possible to make the pixelpipe stop and reprocess if the button is toggled.
